src/backend-service-connector/config.tsx

```diff
@@ -1,15 +1,15 @@
 export class Config {
-    static readonly baseUsersUrl = 'http://localhost:8080/users'
-    static readonly getAllUsersPath = '/get-all'
-    static readonly removeUserPath = `?login=`
-    static readonly logInUserPath = '/log-in'
-    static readonly editUserEmailPath = '/new-email'
-    static readonly editUserLoginPath = '/new-login'
+  static readonly baseUsersUrl = 'http://localhost:8080/users'
+  static readonly getAllUsersPath = '/get-all'
+  static readonly logInUserPath = '/log-in'
+  static readonly editUserEmailPath = '/new-email'
+  static readonly editUserLoginPath = '/new-login'
 
-    static readonly baseTransactionsUrl = 'http://localhost:8080/transactions'
-    static readonly getAllTransactionsPath = '/get-all'
-    static readonly getAllUserTransactionsPath = '/all-transactions'
-    /**
-     * w klasie config bedziemy przetrzymywali sciezki do fetch aby w razie konieczności edytowac je w jednym miejscu i dla przejzystości kodu
-     * **/
+  static readonly baseTransactionsUrl = 'http://localhost:8080/transactions'
+  static readonly getAllUserTransactionsPath = '/all-transactions'
+  static readonly getAllCoinsInfoPath = '/get-coins-prices'
+  static readonly deleteTransactionPath = '/delete-transaction?transactionId='
+  static readonly getUserCoinPath = '/get-coin?name='
+
+  static readonly baseWalletUrl = 'http://localhost:8080/wallet'
```

Comment:
for safety, it should be kept in [.env file](https://create-react-app.dev/docs/adding-custom-environment-variables/)